app/templates/settings.html

```diff
@@ -182,7 +182,8 @@ <h2>{{ _('About groceri.es') }}</h2>
 function profile() {
   let form = document.getElementById('profile-form')
   form.addEventListener('submit', function(event) {
-    let uri = '{{ url_for('profile') }}'
```

Comment:
This was causing the settings page to error-out completely. I see that there was already an issue with the settings page, which I'm guessing is because there isn't a profile page or a profile.html template. So, I'm thinking the best path forward is to create the page or template?